src/Templating/Helper/UploaderHelper.php

```diff
@@ -9,20 +9,8 @@
 
 class UploaderHelper extends Helper
```

Comment:
The Templating component is [deprecated in Symfony 6.0](https://github.com/symfony/symfony/blob/aeb248933cd1a31b96531e1999cd97159bf0c493/UPGRADE-7.0.md#templating) and will be removed in 7.0.

I think this UploadHelper will need to change a bit more, though I'm unsure how. Do we even need to extend it? All it does is provide a charset.

I opened #437 to track this.